src/Elastic.Apm/BackendComm/BackendCommUtils.cs

```diff
@@ -216,19 +216,27 @@ private static HttpClientHandler CreateHttpClientHandler(IConfiguration configur
 			// and runtime exceptions on .NET Framework versions <4.7.2, we don't attempt to set the certificate
 			// validation callback or SSL protocols on net462, which should also apply to .NET Framework <4.7.2 runtimes
 			// which resolve to that target.
+#if NET3_0_OR_GREATER
+			httpClientHandler.SslProtocols |= SslProtocols.Tls12;
```

Comment:
I do wonder if we should just let this default to `SslProtocols.None` at least for .NET 8 and up. 

The defaults are good out of the box in .NET 

https://learn.microsoft.com/en-us/dotnet/api/system.security.authentication.sslprotocols?view=net-9.0

Reply:
Makes sense to me. @JeremyBessonElastic - Would you be okay to update and then I think we can merge.